lib/encoder/src/Pololu37D.cpp

```diff
@@ -5,23 +5,55 @@
 using namespace Encoder;
 
 Pololu37D::Pololu37D(const Config &config)
-    : m_QEI(config.ChannelA, config.ChannelB, config.Index, config.quadratureEncoding),
+    : m_QEI(config.ChannelA, config.ChannelB, config.Index, GPIO::QEI::Encoding::X4_ENCODING),
       m_zeroOffsetDeg(config.offsetDeg) {}
 
 bool Pololu37D::getAngleDeg(float &angle) {
   std::scoped_lock<Mutex> lock(m_mutex);
-  angle = (m_QEI.getPulses() * m_degreesPerCount) - m_zeroOffsetDeg;
+  if (read()) {
```

Comment:
no need for the check since ur internal read always returns true

Reply:
I agree, done